lib/Backend/Func.h

```diff
@@ -546,6 +567,7 @@ static const unsigned __int64 c_debugFillPattern8 = 0xcececececececece;
     bool                hasBailout: 1;
     bool                hasBailoutInEHRegion : 1;
     bool                hasStackArgs: 1;
+    bool                hasImplicitParamLoad : 1;
```

Comment:
>hasImplicitParamLoad [](start = 24, length = 20)

nit: hasImplicitParamLoadFromStack. You can add a comment, that is indicates the load of callInfo and FunctionObject from stack.